docs/products/kafka/karapace/getting-started.rst

```diff
@@ -0,0 +1,12 @@
+Getting Started with Karapace
+=============================
+
+To use Karapace, you need to enable **Karapace Schema registry** and **REST APIs** on your Aiven for Apache Kafka® service.  This can be enabled through the Aiven consol.
```

Comment:
Should we have a tip to say that you don't need BOTH. Depending on the functionality you're looking after only one of them could be sufficent

Reply:
Mentioned this in step 2.